stories/modules-tagmanager-setup.stories.js

```diff
@@ -21,37 +21,22 @@
  */
```

Comment:
Similar to AdSense, something here seems to mess up several Tag Manager setup stories, e.g.:
<img width="1512" alt="Screenshot 2020-10-12 at 15 44 19" src="https://user-images.githubusercontent.com/3531426/95796523-f08d6b80-0ca1-11eb-917a-ca0292a62581.png">